<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lobbyUI</class>
 <widget class="QWidget" name="lobbyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="NameQString">
       <property name="text">
        <string>userName</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="ipNum">
       <property name="text">
        <string>ipNum</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ipEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="portNum">
       <property name="text">
        <string>portNum</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="sendMesgEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="join_server">
     <property name="text">
      <string>Join Server</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="goto_prod">
     <property name="text">
      <string>Goto prod</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
